lib/core/src/display/shape/text/glyph/font.rs

```diff
@@ -260,61 +254,58 @@ mod tests {
     wasm_bindgen_test_configure!(run_in_browser);
 
     #[wasm_bindgen_test(async)]
-    fn empty_font_render_info() -> impl Future<Output=()> {
-        TestAfterInit::schedule(||{
-            let font_render_info = create_test_font_render_info();
-
-            assert_eq!(TEST_FONT_NAME, font_render_info.name);
-            assert_eq!(0, font_render_info.msdf_texture.data.len());
-            assert_eq!(0, font_render_info.glyphs.len());
-        })
+    async fn empty_font_render_info() {
+        basegl_core_msdf_sys::initialized().await;
+        let font_render_info = create_test_font_render_info();
+
+        assert_eq!(TEST_FONT_NAME, font_render_info.name);
+        assert_eq!(0, font_render_info.msdf_texture.borrow().data.len());
+        assert_eq!(0, font_render_info.glyphs.borrow().len());
```

Comment:
All these things should have safe wrappers. You can still keep code herel ike `font_render_info.glyphs.len()` even when using borrow under the hood. The previous API was nice. I feel this is worse than our standards here.